test/index.js

```diff
@@ -1000,6 +1000,20 @@ describe('options.baseUrl', () => {
         expect(promise.req._headers.host).to.equal('localhost:8080');
     });
 
+    it('uses lower-case host header when path is not a full URL', async () => {
+
+        const promise = Wreck.request('get', '/foo', { baseUrl: 'http://localhost:0/', headers: { Host: 'localhost:8080' } });
```

Comment:
this might be a typo, should it be `host` instead of `Host`

Reply:
Nice catch. Fixed in 932f1fd.